src/olympia/scanners/tasks.py

```diff
@@ -336,7 +336,7 @@ def run_yara_query_rule(query_rule_pk):
     )
     if not rule.run_on_disabled_addons:
         qs = qs.exclude(addon__status=amo.STATUS_DISABLED)
-    qs = qs.values_list('id', flat=True).order_by('pk')
+    qs = qs.values_list('id', flat=True).order_by('-pk')
```

Comment:
Do you think this should be an option when creating the query?

Reply:
No. For a complete result, you need to wait for the task to finish anyway, this just helps me to get a head start.